build/gitkubed/Dockerfile

```diff
@@ -4,10 +4,10 @@ MAINTAINER tiru@hasura.io
 # Install openssh server
 RUN apt-get update \
  && apt-get install -y upx-ucl binutils curl openssh-server git jq \
- && curl -o /tmp/docker-1.12 'https://get.docker.com/builds/Linux/x86_64/docker-1.12.6.tgz' \
- && tar -xf /tmp/docker-1.12 -C /tmp \
+ && curl -o /tmp/docker-18.06 'https://download.docker.com/linux/static/stable/x86_64/docker-18.06.1-ce.tgz' \
+ && tar -xf /tmp/docker-18.06 -C /tmp \
  && mv /tmp/docker/docker /bin/docker \
```

Comment:
For some reason the server isn't upgrading even after changing the potentially buggy L8. I changed L8 to `cp /tmp/docker/* /bin/` to copy all the binaries including dockerd yet: 
```
root@gitkubed-6cf7744876-jwg2w:/# docker version
Client:
 Version:           18.06.1-ce
 API version:       1.27 (downgraded from 1.38)
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:20:43 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          17.03.2-ce
  API version:      1.27 (minimum version 1.12)
  Go version:       go1.9.2
  Git commit:       f5ec1e2
  Built:            Fri Aug 10 11:50:21 2018
  OS/Arch:          linux/amd64
  Experimental:     false
```

Reply:
Isn't the container just taking to the host daemon (through a bind-mounted `/var/run/docker.sock`? The Dockerfile only installs the docker cli, not the daemon

Reply:
Yes! Completely missed it :) Hmm, so no way to solve this if cloud providers don't give us newer docker.